crates/sp-domains/src/fraud_proof.rs

```diff
@@ -225,6 +225,22 @@ pub struct InvalidStateTransitionProof {
     pub execution_phase: ExecutionPhase,
 }
 
+pub fn dummy_invalid_state_transition_proof(
```

Comment:
Why not move this function into the pallet-domains benchmarking module similar to `create_dummy_receipt`, it's only used there.